templates/components/products/product-view.html

```diff
@@ -28,6 +29,7 @@ <h2 class="productView-brand"{{#if schema}} itemprop="brand" itemscope itemtype=
                 {{/or}}
             </div>
             {{{region name="product_below_price"}}}
+            {{#if product.rating}}
```

Comment:
This seems unrelated to the change. Is there another bug you are trying to fix?

Reply:
yes we were getting rating sections showing up on products without ratings

Reply:
Ok, with the introduction of this new block, please indent the contents